nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-site-to-site/src/test/groovy/org/apache/nifi/remote/StandardPublicPortGroovyTest.groovy

```diff
@@ -109,6 +98,6 @@ class StandardPublicPortGroovyTest extends GroovyTestCase {
         logger.info("Received ${responses.sum()} total flowfiles")
 
         // Assert
-        assert !responses.isEmpty()
+        assertFalse(!responses.isEmpty())
```

Comment:
```suggestion
        assertFalse(responses.isEmpty())
```

Need to remove the `!` to retain the original assertion logic (or use `assertTrue` with the `!` but that would be more confusing to maintain)

Reply:
@ChrisSamo632 Good catch!